iotedgedev/envvars.py

```diff
@@ -209,6 +207,25 @@ def save_envvar(self, key, value):
             self.output.error(f("Could not update the environment variable {key} in file {dotenv_path}"))
             sys.exit(-1)
 
+    def get_registries(self):
+        registries = {}
+        self.CONTAINER_REGISTRY = {}
+        for key in os.environ:
+            if key.startswith('CONTAINER_REGISTRY_SERVER'):
+                token = key[25:]
+                if token not in registries:
+                    registries[token] = {}
+                registries[token]['server'] = os.environ[key]
+            elif key.startswith(('CONTAINER_REGISTRY_USERNAME', 'CONTAINER_REGISTRY_PASSWORD')):
+                token = key[27:]
+                value = key[19:27].lower()
+                if token not in registries:
+                    registries[token] = {}
+                registries[token][value] = os.environ[key]
+
+        for key, value in registries.items():
+            self.CONTAINER_REGISTRY[key] = ContainerRegistry(value['server'], value['username'], value['password'])
```

Comment:
Looks good. Do we need to assign the old properties for any back-compatibility or will all usages be internal, and dig into this for the registry w/default token? I'm thinking
```
defaultRegistry = registries['']
self.CONTAINER_REGISTRY_SERVER = defaultRegistry['server']
...
```